Bottom: be1d148f40c29b47de60fb983dc385cfd8e431c2
Top:    a2dfd2ffb8e097031065bacf51f68a4fd276644d
Author: Parth Shah <parth@linux.ibm.com>
Date:   2019-07-09 11:42:03 +0530

Refresh of sched-topology-introduce

---

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index fd90633cda72..6784dff0bd0c 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -992,6 +992,8 @@ static inline void update_idle_core(struct rq *rq)
 		__update_idle_core(rq);
 }
 
+extern __read_mostly int sched_smt_weight;
+
 #else
 static inline void update_idle_core(struct rq *rq) { }
 #endif
